response.go

```diff
@@ -44,8 +46,25 @@ func checkResponse(resp *http.Response, headerCondition func(string) Issues) (he
 // `header` is `nil`.
 // To interpret `issues`, see the list of conventions in the
 // documentation for Issues.
-func PreloadableResponse(resp *http.Response) (header *string, issues Issues) {
-	return checkResponse(resp, PreloadableHeaderString)
+func PreloadableResponse(resp *http.Response, policy preloadlist.PolicyType) (header *string, issues Issues) {
+	return checkResponse(resp, PreloadableHeaderStringWrapper, policy)
+}
```

Comment:
This function signature shouldn't change - if someone wants to call PreloadableResponse with a policy, they should call EligibleResponse instead.